components/Map/Map.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import L from 'leaflet';
+
+interface MapProps {
+  saveMapState: (arg0: any, arg1: any) => void;
```

Comment:
What's arg0 and arg1? Give proper names ... and definitely not `any` as a type.

Reply:
Updated and pushed

Reply:
Removed the map labels and turned boundaries to white.